packages/core/src/UserState.ts

```diff
@@ -238,11 +238,15 @@ export default class UserState extends Synchronizer {
         })
     }
 
-    async getAttestations(epochKey: string): Promise<IAttestation[]> {
+    async getAttestations(
+        epochKey: string,
+        epoch: number
+    ): Promise<IAttestation[]> {
```

Comment:
why not put it in `synchronizer`? It is public data

Reply:
Ah I don't think it matters one way or another. This function is removed in 2.0.0. We just need to fix it because it's used in `genUserStateTransition` in 1.0.0

Reply:
Or let's put it in `Synchronizer`
If somehow the backend synchronizer needs this feature 😅

Reply:
ok [b8d926f](https://github.com/Unirep/Unirep/pull/237/commits/b8d926f9ec29f7be62226124110157d114449438) if this is good i'll push the 1.0.3 release tomorrow.